src/agents.py

```diff
@@ -1,4 +1,9 @@
 from typing import Annotated, Any, Dict, Sequence, TypedDict
+import os
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()
```

Comment:
What's the motivation here?

Reply:
Once we add more API keys it would be difficult to manage. A .env file is simpler